lib/ansible/modules/cloud/openstack/os_server_tag.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2018, Artem Goncharov <artem.goncharov@gmail.com>
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: os_server_tag
+short_description: Manage server tags
+extends_documentation_fragment: openstack
+version_added: "2.8"
+author: "Artem Goncharov (gtema)"
+description:
+    - set or delete tag on the Nova server
+options:
+   server:
+      description:
+        - Name or id of the server
+      required: true
+   state:
+     description:
+       - Should the resource be present or absent.
+     choices: [present, absent]
+     default: present
+   tag:
+     description:
+       - tag name
+     required: true
+   availability_zone:
+     description:
+       - Ignored. Present for backwards compatibility
+requirements:
+    - "python >= 2.7"
+    - "openstacksdk"
+'''
+EXAMPLES = '''
+---
+- name: set the tag on the instance
+  ignore_errors: True
+  os_server_tag:
+    server: "{{ server_name }}"
+    state: present
+    name: __crazy_tag
+'''
+
+RETURN = '''
+server:
+    description: UUID of the server.
+    returned: success
+    type: string
+changed:
```

Comment:
this is a common return and must not be documented in a module, please remove it.

Reply:
Done